<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoRaTerminal</class>
 <widget class="QMainWindow" name="LoRaTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoRaTerminal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="uarticon.qrc">
    <normaloff>:/new/prefix1/Resources/6_96px_1168279_easyicon.ico</normaloff>:/new/prefix1/Resources/6_96px_1168279_easyicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>791</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>日志输出</string>
    </property>
    <widget class="QTextEdit" name="textEditLogPrintf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>771</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_savelog">
        <property name="text">
         <string>日志保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clearlog">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSpinBox" name="spinBox_CMDtime">
     <property name="geometry">
      <rect>
       <x>491</x>
       <y>10</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>指令时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>读写地址</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_JzqNo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;显示/输入:7个字节通讯地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maxLength">
      <number>14</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ReadjzqNo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;读取终端采集器地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>读地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_WritejzqNo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>写地址</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>抄表</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_MT_Data">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>表读数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>m³</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>表状态：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtoBreadMeterButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>点对点抄表</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Volatage">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>电压：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>版本号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MT_ver">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Protocol_ver">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>表类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>协议号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DQ_DJL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>月冻结量：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BtoBmeterNo_For_ReadMT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入点对点抄表地址！</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MT_Status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>信号强度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RSSI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>211</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>数据同步</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_BtoBmeterNo_For_sys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa007f;&quot;&gt;输入4字节表地址，例如：12345678&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入点对点同步数据表地址！</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BtoBSaveNo_For_sys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa007f;&quot;&gt;输入2个字节存储地址：例如:1115&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入该表在终端的存储地址！</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_BToBDataSys">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>点对点数据同步</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>元</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>元</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>78</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>结算单价：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PriceMoney">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>78</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa007f;&quot;&gt;例如：2.54元&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>50</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>结算余额：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BalanceMoney">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;例如：25.54元&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>22</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>结算时间：</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Balance">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>22</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
     <property name="date">
      <date>
       <year>2018</year>
       <month>1</month>
       <day>5</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>106</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>阀门控制：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Gatecont_SYS">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>106</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar_BtoB">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>270</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>88</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="uarticon.qrc"/>
 </resources>
 <connections/>
</ui>
